qubesadmin/tools/qvm_device.py

```diff
@@ -303,10 +424,4 @@ def main(args=None, app=None):
 
 
 if __name__ == '__main__':
-    # Special treatment for '--list-device-classes' (alias --list-classes)
-    curr_action = sys.argv[1:]
-    if set(curr_action).intersection(
-            {'--list-device-classes', '--list-classes'}):
-        sys.exit(main(args=['', '--list-device-classes']))
-
```

Comment:
What happened to this one (besides it being already slightly broken due to requiring a device class anyway)?

Reply:
I slightly fix it and now device class is not required :D  (see lines 408-411).